packages/web/src/scss/components/UNSTABLE_ProductLogo/README.md

```diff
@@ -0,0 +1,63 @@
+# UNSTABLE ProductLogo
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+The ProductLogo component is used to display the logo of the product.
+
+```html
+<div class="UNSTABLE_ProductLogo" aria-label="Logo of the product">
+  <!-- Logo go here -->
+</div>
+```
+
+## Inverted variant
+
+```html
+<div class="UNSTABLE_ProductLogo UNSTABLE_ProductLogo--inverted" aria-label="Logo of the product">
+  <!-- Inverted logo go here -->
+</div>
+;
+```
+
+## Content
+
+The content of the ProductLogo component can be an image or svg.
+
+### Image
+
+```html
+<div class="UNSTABLE_ProductLogo" aria-label="Logo of the product">
+  <img src="path-to-logo" alt="Product Logo" aria-hidden="true" />
+</div>
+```
+
+ℹ️ Don't forget to add the `aria-label` attribute for accessible title.
```

Comment:
Ohh, it would be really nice to mention `width` and `height` attributes as well (same for Avatar component), but let's write it down to the list of the tasks for component stabilisation. 🙏🏻